core/src/main/scala/org/apache/spark/internal/io/SparkHadoopMapReduceWriter.scala

```diff
@@ -119,7 +119,7 @@ object SparkHadoopMapReduceWriter extends Logging {
     }
   }
 
-  /** Write a RDD partition out in a single Spark task. */
+  /** Write an RDD partition out in a single Spark task. */
```

Comment:
Yes, "an RDD" is correct because "a" precedes consonant _sounds_ and "an" precedes vowel sounds. So it's "an _arr-dee-dee_".

We don't need to fix up every last instance of this. It was a good idea to fix it all while fixing some that overlapped with this PR though.